src/app/shared/components/panels/panels.component.html

```diff
@@ -1,17 +1,25 @@
 <div *ngIf="curWidth >= 960" class="full-width">
   <div *ngFor="let key of getKeyList(placements)">
     <h2 *ngIf="placements[key].topTitle"
-        [style.color]="placements[key].topTitle.titleColor ? placements[key].topTitle.titleColor : null" class="top-title">
+        [style.color]="placements[key].topTitle.titleColor ? placements[key].topTitle.titleColor : null"
+        class="top-title">
       {{placements[key].topTitle.title}}</h2>
     <div fxLayout="row"
          fxLayoutAlign="start start">
       <div [fxFlex]="getRowParts(im)" *ngFor="let im of placements[key].imgs" class="container">
         <a (click)="directToUrl(im.href)">
-          <img [src]="im.imgUrl" class="full-width"/>
+          <img *ngIf="getFileTypeFromExtension(im.fileType && im.fileType['ext'], getImageLink(im.imgUrl)) === 'image'"
```

Comment:
calling function in template is not a good practice - please define a public variable and use it instead (like 

```
isVideo = getFileTypeFromExtension(im.fileType && im.fileType['ext'], getImageLink(im.imgUrl)) === 'video';
isImage = getFileTypeFromExtension(im.fileType && im.fileType['ext'], getImageLink(im.imgUrl)) === 'image';
```